packages/wonder-blocks-spacing/docs.md

```diff
@@ -1,42 +1,93 @@
 Spacing is a collection of simple names assigned to specific dimensions. These are used frequently when laying out Wonder Blocks components (such as with the Grid). You can use these sizes directly by importing the `wonder-blocks-spacing` package and accessing the named property like so: `Spacing.xxSmall`.
 
 ```js
+const {StyleSheet} = require("aphrodite");
+const {View} = require("wonder-blocks-core");
 const Spacing = require("./index.js").default;
+const {Strut} = require("./index.js");
 
-<div>
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+        alignItems: "center",
+        marginBottom: Spacing.xxSmall,
+    },
+});
+
+<View>
     {Object.keys(Spacing).map((spaceName, idx) => (
-        <div
+        <View
```

Comment:
I want to make sure that all of our docs use `View` instead of `div` to help encourage people to use `View`.